.github/workflows/docker.yml

```diff
@@ -48,7 +51,24 @@ jobs:
             echo "tag=all" >> "$GITHUB_OUTPUT"
           fi
 
+      - name: Create tags for Docker images based on build-time arguments (release)
+        if: github.event_name == 'release' && github.event.action == 'published'
+        id: tags-release
+        run: |
+          if [ "${{ matrix.build-args }}" = "No solvers" ]; then
+            echo "tag=${{ github.event.release.name }}-latest" >> "$GITHUB_OUTPUT"
```

Comment:
I am not sure yet if `github.event.release.name` will work, we might need to [use github.ref_name` instead](https://github.com/orgs/community/discussions/26686#discussioncomment-3396593) to get the version. 

I will request @arjxn-py to uncomment the `if:` condition at the top, comment out the login step, change `push: true` to `push: false` and check the tag of the built image manually.

Reply:
I've done this in https://github.com/arjxn-py/PyBaMM/actions/runs/7350486868, as defined it is taking in release name (i.e. `Test Release` in this case) which looks alright but it would be nicer to have tag instead, hopefully the one you suggested https://github.com/orgs/community/discussions/26686#discussioncomment-3396593 do the work

Reply:
Yes now it takes in tags in - https://github.com/arjxn-py/PyBaMM/actions/runs/7350650963/job/20012681428